djorg/djorg/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for djorg project.
+
+Generated by 'django-admin startproject' using Django 2.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'w_764@wmp08eb3p#!a%mbo!kq7!gcvq!(#d3&188-zo2b^r8+w'
+
```

Comment:
Your secret key should be in your .env file, accessible by using python-decouple as instructed in Day 1.